### DISCUSSION TITLE
Argo App Health Doesn't change

### DISCUSSION THREAD

**Discussion by anggutie-dev** [argoproj/argo-cd#9646] [Q&A]
Hello All,

I have some jobs that are being deployed/managed by Argo. Some of them fail from time to time. I would like to know why the Argo App Health doesn't change accordingly

![image](https://user-images.githubusercontent.com/89117906/173415914-5b3f7a24-2634-4928-8d77-231a2795028f.png)

Is there a way to make Argo Notifications take those failed jobs into account and send a notification?

**Comment by verb5** [comment]
> Hello All,
> 
> I have some jobs that are being deployed/managed by Argo. Some of them fail from time to time. I would like to know why the Argo App Health doesn't change accordingly
> 
> ![image](https://user-images.githubusercontent.com/89117906/173415914-5b3f7a24-2634-4928-8d77-231a2795028f.png)
> 
> 
> Is there a way to make Argo Notifications take those failed jobs into account and send a notification?

Did you find an answer to your question? I would like to know the same
